tests/test_flash_attn.py

```diff
@@ -17,6 +19,31 @@
 from flash_attn.flash_attn_interface import _get_block_size_n
 from flash_attn.layers.rotary import apply_rotary_emb
 
+# this ensures that the same config will always get the same result
+REPRODUCIBLE=True
+if REPRODUCIBLE:
+    skip_seed = 42
+else:
+    skip_seed = time.time()
+random.seed(skip_seed)
+
+def is_power_of_2(n):
+    return n > 0 and (n & (n - 1)) == 0
+
+def skip_config(*args,  skip_pct = 0.95):
+    return random.random() >= (1.0 - skip_pct)
+
+def skip_not_power_of_2_config(*args):
+    if not all(is_power_of_2(arg) for arg in args):
+        return True
+    else:
+        return False
+
+def is_amd():
```

Comment:
Can we rename this to `is_hip()`, so be consistent with other places?

Reply:
done